<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveRoutineWindow</class>
 <widget class="QDialog" name="SaveRoutineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="text_layout">
    <item>
     <widget class="QLabel" name="text">
      <property name="text">
       <string>Save Routine As: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_box"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add_button">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
